Whiskey/en-US/about_Whiskey_Tasks.help.txt

```diff
@@ -13,6 +13,8 @@ LONG DESCRIPTION
 
     * `CopyFile`: copies files.
     * `Delete`: deletes files.
+    * `DotNetBuild`: runs `dotnet build` command.
+    * `DotNetPack`: runs `dotnet pack` command.

```

Comment:
Commit message says you wrote a test to ensure all tasks are included in this file but I don't see the test, just these additions to `about_Whiskey_Tasks.help.txt`.

Reply:
Forgot to add it. Fixed.